core/systems/assignment/gdpr_form.ex

```diff
@@ -71,15 +91,32 @@ defmodule Systems.Assignment.GdprForm do
   end
 
   @impl true
-  def handle_event("update", %{status: :off}, %{assigns: %{entity: assignment}} = socket) do
-    {:ok, _} = Assignment.Public.update_consent_agreement(assignment, nil)
-
+  def handle_event("update", %{status: :off}, socket) do
     {
       :noreply,
```

Comment:
@MelchiorKokernoot Klein dingetje nog: ik mis hier nog een check op body is gelijk aan default body text (zie gettext key "default.consent.text" in eyra-consent.po). De consent input heeft itt tot de andere inputs een default value. Zonder deze check zou je dus telkens de modal zien ook al heb je zelf niks aan de text aangepast. Mee eens?